src/tests/ftest/io/ior_hdf5.yaml

```diff
@@ -0,0 +1,48 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - server-C
+    - server-D
+    - client-E
+    - client-F
+    - client-G
+    - client-H
+timeout: 3600
+server_config:
+    name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:5e:00.0","0000:5f:00.0"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 1TiB
+    nvme_size: 1TiB
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on
+    control_method: daos
+hdf5_vol:
+    plugin_path: /usr/lib64/mpich/lib
+ior:
+    client_processes:
+        np_72:
+            np: 72
+    test_file: daos:testFile
+    repetitions: 1
+    dfs_destroy: True
+    iorflags:
+        flags: '-v -w -W -r -R'
+    api: HDF5
+    transfer_size: '1G'
+    block_size: '26G'
```

Comment:
this will go to nvme. we are creating 26G x 72 file:
1872 GB which is more than your pool size.